### PR TITLE
Login page UI

### PR DESCRIPTION
**Pull request by Twdckcv**
Login UI is done

### CODE REVIEW

File: frontend/src/themes/style/style.css:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+ReferrerUrl=C:\Users\rubai\OneDrive\Desktop\themeforest-11446068-materialize-material-design-admin-template.zip
```
Comment (The-TT-Hacker) [inline frontend/src/themes/style/style.css:Zone.Identifier]: why is there an absolute path for your system?

Comment (The-TT-Hacker) [reply frontend/src/themes/style/style.css:Zone.Identifier]: check the other files for this as well

**Comment by The-TT-Hacker** [comment]
We are going to need an end to end test for this to make the pipeline pass

File: frontend/src/themes/style/style.css:Zone.Identifier (hunk 1 above)
Comment (Twdckcv) [reply frontend/src/themes/style/style.css:Zone.Identifier]: I have removed all of that

File: frontend/src/index.tsx [hunk 2]
```diff
@@ -7,6 +7,8 @@ import reportWebVitals from './reportWebVitals';
 import { createRoot } from 'react-dom/client';
 import ReactDOM from 'react-dom/client';
 import { BrowserRouter } from 'react-router-dom';
+import GitHubLogin from './pages/githublogin';
```
Comment (The-TT-Hacker) [inline frontend/src/index.tsx:10]: the filename should be GithubLogin.tsx as all of the other components

File: frontend/src/pages/githublogin.tsx [hunk 3]
```diff
@@ -1,11 +1,37 @@
...
 
 const GitHubLogin = () => {
   return (
-    <div>
-      <p>Login with Github</p>
-    </div>
+    <html>
+      <body className="login-background">
+        <div id="login-page" className="row">
+          <div className="col s12 z-depth-4 card-panel">
```
Comment (The-TT-Hacker) [inline frontend/src/pages/githublogin.tsx:14]: should use react-bootstrap instead of using the css directly. it gives you Col and Row components

**Pull request closed without merging**